cmd/policy/policy.go

```diff
@@ -134,9 +134,62 @@ func NewCommand(config *settings.Config, preRunE validator) *cobra.Command {
 		return cmd
 	}()
 
+	update := func() *cobra.Command {
+		var policyPath string
+		var policyID string
+		var updateRequest policy.UpdateRequest
+
+		cmd := &cobra.Command{
+			Short: "Update a policy",
+			Use:   "update",
+			RunE: func(cmd *cobra.Command, args []string) error {
+				policyData, err := os.ReadFile(policyPath)
+				if err != nil {
+					return fmt.Errorf("failed to read policy file: %w", err)
+				}
+
+				updateRequest.Content = string(policyData)
+
+				client := policy.NewClient(*policyBaseURL, config)
+
+				result, err := client.UpdatePolicy(*ownerID, policyID, updateRequest)
+				if err != nil {
+					return fmt.Errorf("failed to update policy: %w", err)
+				}
+
+				enc := json.NewEncoder(cmd.OutOrStdout())
+				enc.SetIndent("", "  ")
+
+				if err := enc.Encode(result); err != nil {
+					return fmt.Errorf("failed to encode result to stdout: %w", err)
+				}
+
+				return nil
+			},
+			Args:    cobra.ExactArgs(0),
+			Example: `policy update --owner-id 462d67f8-b232-4da4-a7de-0c86dd667d3f --policy-id e9e300d1-5bab-4704-b610-addbd6e03b0b --name policy_name --active --policy ./policy.rego`,
```

Comment:
policy-id as the resource ID would be a good candidate for positional:

Example:
```
circle policy update some-resource-uuid-string --name new-name
```

Reply:
since we also require `owner-id` as a non-positional flag I think it makes sense to make `policy-id` non-positional too. Also reduces confusion since we have two uuids - easy to mix up

Reply:
It would only be confusing if the both were unflagged. I think we should rely on prior art here in  CLI design. Examples include kubectl, when working on the resource the id is positional:

```
kubectl describe pod {PodID} --flags...
docker container rm {containerID} --flags...

circle-cli policy update {policyID} --flags ...
```

Reply:
those are both examples of commands with only a single UUID, so there's never any confusion. If anything, the org id should have been positional since it's required for every policy command